job/src/main/resources/index.jelly

```diff
@@ -25,5 +25,5 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <div>
-    Defines a new job type for workflows and provides their generic user interface.
+    Defines a new job type for pipelines and provides their generic user interface.
```

Comment:
@kwhetstone I'm indifferent on a many of these. Though it could really go either way, in this case we're not defining a job type to create an arbitrary pipeline, we're specifically creating a Jenkins Pipeline.
Although "Pipeline creates pipelines", referring to specific items that the plugin creates, uses, or is otherwise directly tied to it should be referred to as "Pipeline(s)", even though they might also describe a generic pipeline. I didn't bug or ant this, though; it is not that important. :)